contrib/vagrant/start.sh

```diff
@@ -613,8 +613,12 @@ function createVm(){
     else
         vagrant up $PROVISION_ARGS $1
         if [ "$?" -eq "0" -a -n "${K8S}" ]; then
-            host_port=$(vagrant port --guest 6443 k8s1)
-            vagrant ssh k8s1 -- cat /home/vagrant/.kube/config | sed "s;server:.*:6443;server: https://k8s1:$host_port;g" > vagrant.kubeconfig
+            hostname=k8s1
+            if [ ! -z "$NETNEXT" -a "$NETNEXT" = "true" -o "$NETNEXT" = "1" ]; then
+                hostname=k8s1+
+            fi
+            host_port=$(vagrant port --guest 6443 $hostname)
+            vagrant ssh $hostname -- cat /home/vagrant/.kube/config | sed "s;server:.*:6443;server: https://k8s1:$host_port;g" > vagrant.kubeconfig
             echo "Add '127.0.0.1 k8s1' to your /etc/hosts to use vagrant.kubeconfig file for kubectl"
```

Comment:
nit : `k8s1` is hardcoded here.

Reply:
@aditighag See the pull request description. This is not updated on purpose. It's still k8s1 in the VM.